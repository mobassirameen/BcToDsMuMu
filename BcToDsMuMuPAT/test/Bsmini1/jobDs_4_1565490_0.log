000 (1565490.000.000) 04/06 13:40:24 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_1694_8585_240>
...
040 (1565490.000.000) 04/06 13:44:03 Started transferring input files
	Transferring to host: <128.142.64.34:9618?addrs=128.142.64.34-9618+[2001-1458-301-4c--100-597]-9618&noUDP&sock=2575_1202_45955>
...
040 (1565490.000.000) 04/06 13:44:03 Finished transferring input files
...
001 (1565490.000.000) 04/06 13:44:04 Job executing on host: <128.142.64.34:9618?addrs=128.142.64.34-9618+[2001-1458-301-4c--100-597]-9618&noUDP&sock=2355_e99f_3>
...
006 (1565490.000.000) 04/06 13:44:08 Image size of job updated: 12808
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
040 (1565490.000.000) 04/06 13:44:08 Started transferring output files
...
040 (1565490.000.000) 04/06 13:44:08 Finished transferring output files
...
005 (1565490.000.000) 04/06 13:44:08 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	24355  -  Run Bytes Sent By Job
	16042  -  Run Bytes Received By Job
	24355  -  Total Bytes Sent By Job
	16042  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        4         4 
	   Disk (KB)            :      537    15000    275681 
	   Memory (MB)          :        0     8000      8000 
...
