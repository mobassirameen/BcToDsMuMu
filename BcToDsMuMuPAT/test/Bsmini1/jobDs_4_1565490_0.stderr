df: ‘/eos/atlas’: Connection timed out
df: ‘/eos/lhcb’: Connection timed out
which: no srmcp in (/afs/cern.ch/cms/caf/scripts:/cvmfs/cms.cern.ch/common:/afs/cern.ch/cms/caf/scripts:/cvmfs/cms.cern.ch/common:/usr/sue/bin:/bin:/usr/condabin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/opt/puppetlabs/bin)
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
condor_exec.exe: line 60: cd: CMSSW_10_2_12/src/bctodsmumu_analysis/BcToDsMuMuPAT/test/: No such file or directory
SCRAM fatal: Unable to locate the top of local release. Please run this command from a SCRAM-based area.
SCRAM fatal: Unable to locate the top of local release. Please run this command from a SCRAM-based area.
mv: cannot stat ‘../../../../../BcToDsMuMuRootupler.py’: No such file or directory
SCRAM fatal: Unable to locate the top of local release. Please run this command from a SCRAM-based area.
which: no cmsRun in (/afs/cern.ch/cms/caf/scripts:/cvmfs/cms.cern.ch/common:/afs/cern.ch/cms/caf/scripts:/cvmfs/cms.cern.ch/common:/usr/sue/bin:/bin:/usr/condabin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/opt/puppetlabs/bin)
xrdcp: no such file or directory processing BctoDsMuMu_MC_4.root
